test/new-e2e/utils/credentials/store-aws.go

```diff
@@ -21,7 +21,8 @@ func (s *awsStore) get(key string) (string, error) {
 		return "", err
 	}
 
-	output, err := ssmClient.GetParameter(context.Background(), &ssm.GetParameterInput{Name: &key, WithDecryption: true})
+	withDecription := true
+	output, err := ssmClient.GetParameter(context.Background(), &ssm.GetParameterInput{Name: &key, WithDecryption: &withDecription})
```

Comment:
Is this change intended?

Reply:
Yes, hard to read, but `WithDecription` moved from type `bool` (boolean value) to type `*bool` (boolean pointer).